discovery-provider/src/queries/queries.py

```diff
@@ -131,6 +131,46 @@ def get_tracks():
     return api_helpers.success_response(tracks)
 
 
+# Get all tracks matching a route_id and track_id.
+# Expects a JSON body of shape:
+#   { "tracks": [{ "id": number, "url_title": string, "handle": string }]}
+@bp.route("/tracksIncludingUnlisted", methods=("POST",))
```

Comment:
let's use python snake_case for this route, similar to others

Reply:
will change